packages/components/src/components/SquiggleChart.tsx

```diff
@@ -41,19 +44,40 @@ export const SquiggleChart: FC<SquiggleChartProps> = memo(
       ...(project ? { project, continues } : { environment }),
     });
 
+    // TODO - if `<ViewerProvider>` is not set up (which is very possible) then calculator paths won't be resolved.
+    const getSubvalueByPath = useGetSubvalueByPath();
+
     if (!squiggleOutput) {
       return <RefreshIcon className="animate-spin" />;
     }
 
-    return (
-      <DynamicSquiggleViewer
-        rootPathOverride={rootPathOverride}
-        squiggleOutput={squiggleOutput}
-        isRunning={isRunning}
-        showHeader={showHeader}
-        environment={environment}
-        {...settings}
-      />
-    );
+    if (rootPathOverride) {
+      const rootValue =
+        rootPathOverride.root === "result"
+          ? getResultValue(squiggleOutput)
+          : getResultVariables(squiggleOutput);
+      if (!rootValue) {
+        return <MessageAlert heading="Value is not defined" />;
+      }
+      if (!rootValue.ok) {
+        return <SquiggleErrorAlert error={rootValue.value} />;
```

Comment:
why is this SquiggleErrorAlert, but the others MessageAlert?

Reply:
The name is the same as it was before, but I think it's good:
- this component takes `SqError`, so it's squiggle-specific
- unlike `<MessageAlert>`, this component is public, and other components have `Squiggle` prefix